app/src/main/java/org/schabi/newpipe/fragments/list/BaseListFragment.java

```diff
@@ -344,55 +337,14 @@ protected void onScrollToBottom() {
         }
     }
 
-    protected void showStreamDialog(final StreamInfoItem item) {
+    protected void showInfoItemDialog(final StreamInfoItem item) {
         final Context context = getContext();
         final Activity activity = getActivity();
         if (context == null || context.getResources() == null || activity == null) {
             return;
         }
```

Comment:
This whole function could be removed completely, also in other places where opening the info item dialog uses a single line and is used in only one place. The `null` checks should be moved into the `create()` method of `InfoItemDialog.Builder`, and an error notification should be shown in case anything is `null`.

Reply:
I was also annoyed by the checks, but wasn't sure how to handle them correctly. Showing a snackbar should be good. However, the checks need to be performed in the constructor because the context is used when deciding whether some entries need to be added.

Reply:
Right